tests/validation/suite_test.go

```diff
@@ -0,0 +1,28 @@
+package validation
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/rancher/rio/tests/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.ValidationPreCheck()
+	os.Exit(m.Run())
+}
+
+func TestSuite(t *testing.T) {
+	suite := spec.New("validation suite", spec.Report(report.Terminal{}))
+	specs := map[string]func(t *testing.T, when spec.G, it spec.S){
+		"p1run":    p1AutoscaleTests,
```

Comment:
I wonder if we shouldn't use `p1`, as that is an internal term. Community members have no idea what that means.